packages/core/utils/platform.ts

```diff
@@ -26,10 +20,10 @@ export const runtime: UniversalRuntime = {
 	isBlink,
 	isExtension,
 	isServer,
-	isIOS,
-	isAndroid,
-	isMobile,
-	isBrowser: isWeb && !isExtension && !isServer,
+	isBrowser: !isExtension && !isServer,
+	isMobile: false,
+	isIOS: false,
+	isAndroid: false,
```

Comment:
could we consider removing this `runtime` variable when it does not reflect runtime correctly anymore